lib/vmpooler.rb

```diff
@@ -15,7 +15,10 @@ module Vmpooler
   require 'timeout'
   require 'yaml'
 
-  %w[api graphite logger pool_manager statsd dummy_statsd generic_connection_pool].each do |lib|
+  require 'prometheus/middleware/collector'
```

Comment:
This is what you are replacing, right @jcoconnor ?

Reply:
Removed and tidied up.